x-pack/plugins/security_solution/public/timelines/components/timeline/body/renderers/formatted_field_helpers.tsx

```diff
@@ -91,7 +91,9 @@ export const RenderRuleName: React.FC<RenderRuleNameProps> = ({
   );
   const id = `event-details-value-default-draggable-${contextId}-${eventId}-${fieldName}-${value}-${ruleId}`;
   const link = useMemo(() => {
-    const content = truncate ? (
+    const content = children ? (
```

Comment:
are we sure this would not disrupt other places where this component would be used (it doesn't seem so but checking anyway)?

Reply:
I did check and no other places pass children to this component at all. 

To make it (slightly) cleaner, I added a branch in `RenderRuleName` to check if children is available, if so render children. Because why would you want to render children if it is being passed? Otherwise, it would default to the text value as before.